server/tserver/src/main/java/org/apache/accumulo/tserver/AssignmentHandler.java

```diff
@@ -173,15 +168,7 @@ public void run() {
       tablet = null; // release this reference
       successful = true;
     } catch (Exception e) {
-      log.warn("exception trying to assign tablet {} {}", extent, locationToOpen, e);
-
-      if (e.getMessage() != null) {
-        log.warn("{}", e.getMessage());
-      }
```

Comment:
These lines seemed kinda pointless so I removed them.  Not sure if I am missing something.

Reply:
Removing it looks right to me.